composeApp/src/iosMain/kotlin/org/ooni/probe/background/RunOperation.kt

```diff
@@ -14,7 +16,8 @@ class RunOperation(
 ) : NSOperation() {
     override fun main() {
         Logger.d { "Running operation" }
-        GlobalScope.launch {
+        val coroutineScope = CoroutineScope(SupervisorJob() + Dispatchers.Default)
```

Comment:
This doesn't look related with the iOS issue. Trying to fix background auto-run?